Jenkinsfile

```diff
@@ -10,6 +10,12 @@ def testEnvFolder
 
 def testbeds
 
+def logServer = "logs.opencontrail.org"
```

Comment:
Does jenkinsfile have support for structures or classes ? It would be good to group those variables if it is possible.